python/wheelbuilder/Dockerfile

```diff
@@ -0,0 +1,98 @@
+FROM quay.io/pypa/manylinux1_x86_64
+
+# Configure ld.so/ldconfig and pkg-config
+RUN echo /usr/local/lib64 > /etc/ld.so.conf.d/local.conf && \
+    echo /usr/local/lib >> /etc/ld.so.conf.d/local.conf
+ENV PKG_CONFIG_PATH /usr/local/lib64/pkgconfig:/usr/local/lib/pkgconfig
+
+# Install meson and ninja
+ENV NINJA_VERSION v1.8.2
+RUN /opt/python/cp37-cp37m/bin/pip install meson
+RUN cd /usr/bin && ln -s /opt/_internal/cpython-3.7.0/bin/meson
```

Comment:
in my run this was cpython-3.7.1 -- does it make sense to use python35 for meson which is probably more stable and doesn't see many updates i guess?